### DISCUSSION TITLE
How to get information like limb darkening coefficient after doing the best fit?

### DISCUSSION THREAD

**Discussion by theabraham2000** [hpparvi/PyTransit#154] [General]
After doing the best fit using ta.plot_light_curves(method='fit');
How do we get the quadratic limb darkening coefficients for the plotted best fit light curve?

I was trying to follow this tutorial: https://github.com/hpparvi/PyTransit/blob/master/notebooks/001_simple_broadband_parameter_estimation.ipynb 

(line 11 and line 14)

**Comment by hpparvi** [comment]
Hi Abraham,

Sorry for the delay, you can access the best-fit solution after line 10 from

``ta.de.minimum_location``

but I'd recommend using the posterior distributions that you get after running the sampler (line 16) instead of the best-fit solutions. You can access the posteriors from ``df`` as

``df.posterior.q1_TESS``
``df.posterior.q2_TESS``

Cheers,
Hannu

**Reply by theabraham2000** [reply]
Thank you 👍
